cpp/velox/jni/jni_wrapper.cc

```diff
@@ -15,9 +15,9 @@
  * limitations under the License.
  */
 
-#include <arrow/c/bridge.h>
```

Comment:
The modification here is reasonable. Otherwise, errors will occur when someone else installs other versions of arrow.